recipe/meta.yaml

```
@@ -109,3 +216,4 @@ extra:
     - pstjohn
     - mcs07
     - jaimergp
+    - skearnes
```

```suggestion
    - skearnes
  feedstock-name: rdkit
```

For backwards compat with the previous top-level `package` name.